aspnetcore/blazor/components/prerender.md

```diff
@@ -125,6 +128,14 @@ For components embedded into a page or view of a Razor Pages or MVC app, you mus
 </body>
 ```
 
+## Interactive routing and prerendering
+
+Internal navigation for interactive routing doesn't involve requesting new page content from the server. Therefore, prerendering doesn't occur for internal page requests.
+
+The <xref:Microsoft.AspNetCore.Components.PersistentComponentState> service only works on the initial page load and not across enhanced page navigation events. If the app performs a full (non-enhanced) navigation to a page utilizing persistent component state, the persisted state is made available for the app to use when it becomes interactive. But if an interactive circuit has already been established and an enhanced navigation is performed to a page that renders persisted component state, that state isn't made available in the existing circuit. The <xref:Microsoft.AspNetCore.Components.PersistentComponentState> service isn't aware of enhanced navigation, and there's no mechanism to deliver state updates to components that are already running.
+
+A mechanism to deliver state updates for enhanced navigation is under consideration for .NET 9, which is targeted for release in late 2024. For more information, see [[Blazor] Support persistent component state across enhanced page navigations (dotnet/aspnetcore #51584)](https://github.com/dotnet/aspnetcore/issues/51584). For more information on enhanced navigation, see <xref:blazor/fundamentals/routing#enhanced-navigation-and-form-handling>.
```

Comment:
Should we say this? While I think it's likely we'll do this, there are some unknowns that may impact the strategy/degree to which this gets solved. I appreciate that the wording implies that we haven't fully committed to this yet, but some customers might still misinterpret the meaning. @danroth27 what do you think?

Reply:
(this feedback specifically applies to line 137)

Reply:
I can cut it. Yes, I was trying to say it ***might be*** addressed with the "under consideration for .NET 9" language.

Reply:
Yeah, let's not promise anything for .NET 9. Nothing is committed yet for .NET 9, and even when we "commit" to stuff it often ends up getting cut or punted to a future release.